ts/Core/Series/Series.ts

```diff
@@ -4225,8 +4226,11 @@ class Series {
             typeof options.pointStart !== 'undefined' ||
             typeof options.pointInterval !== 'undefined' ||
             typeof options.relativeXValue !== 'undefined' ||
+            options.dataSorting &&
+                typeof options.dataSorting.enabled !== 'undefined' ||
             options.joinBy ||
             options.mapData || // #11636
+            series.hasOptionChanged('enabled', 'dataSorting') ||
```

Comment:
Do we need this?

Reply:
Changed to `typeof options.dataSorting !== 'undefined'` 👍